lib/rubocop/cop/style/option_hash.rb

```diff
@@ -38,6 +40,16 @@ def suspicious_name?(arg_name)
           cop_config.key?('SuspiciousParamNames') &&
             cop_config['SuspiciousParamNames'].include?(arg_name.to_s)
         end
+
+        def super_used?(node)
+          body = node.parent.body
+
+          if body.begin_type?
+            body.children.any?(&:zsuper_type?)
```

Comment:
`Node#children` contains only direct children. In other words, it does not contains grandchildren.
So the cop still add an offense below.

```ruby
def foo(opts = {})
  if cond
    super
  end
end
```

So we can use `Node#each_node` instead. And probably we can remove `if body.begin_type?` with using the method.
https://github.com/bbatsov/rubocop/blob/9acbec2a22f2dfe1773e63ffda258af50e41db1e/lib/rubocop/ast/node.rb#L233-L263